### ISSUE TITLE
[Accessibility]Luminosity ratio should be greater than or equal to 4.5:1 for container text on “Container.style” page.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6270]
### Target Platforms

iOS

### SDK Version

App Version: 1.0 (1.3.0.20200820.7)

### Application Name

Visualizer

### Problem Description

[29253854](https://microsoft.visualstudio.com/OS/_workitems/edit/29253854)

Luminosity ratio should be greater than or equal to 4.5:1 for container text on “Container.style” page.

### Card JSON

[Container.Style.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.0/Elements/Container.Style.json)

**Comment by almedina-ms** [comment]
Could not repro, closing for now
